rj_gameplay/rj_gameplay/tactic/line_tactic.py

```diff
@@ -22,7 +30,7 @@ def __init__(self, world_state: stp.rc.WorldState):
         for pt in self.move_points:
             # for some reason stp.role doesn't need to be imported?
             self._role_requests.append(
-                (stp.role.cost.PickClosestToPoint(pt), dumb_move.DumbMove)
+                (stp.role.cost.PickClosestRobot(pt), dumb_move.DumbMove)
```

Comment:
hey, I think the code changed under you while you were working, but the costFn's name is now `PickClosestToPoint` (see `stp/role/cost.py`)